### ISSUE TITLE
Misleading GetSSSR description in the documentation

### ISSUE THREAD

**Issue by chmnk** [rdkit/rdkit#5395]
**Describe the bug**
In the [current documentation](https://www.rdkit.org/docs/source/rdkit.Chem.rdmolops.html#rdkit.Chem.rdmolops.GetSSSR) of python's `Chem.GetSSSR`, 
the return value description mistakenly states the the return value is not an `int` (the SSSR size), but a sequence

```
> RETURNS: a sequence of sequences containing the rings found as atom ids
> The length of this will be equal to NumBonds-NumAtoms+1 for single-fragment molecules.
```

which is misleading. It would be nice to 

- change the wording
- explicitly mention `GetSymmSSSR` for those who need a sequence and not the count
- [Feature request] add `FindSSSR` to the python's wrap to have both strict and SymmSSSR ways to compute SSSR.

Also, the function names themselves are misleading since `GetSSSR` returns an `int`, and `GetSymmSSSR` returns a sequence of tuples, but I guess that changing them would spoil the backward compatibility.

**Configuration (please complete the following information):**
 - 2022.03.1pre

**Comment by greglandrum** [comment]
I agree that the return values should be consistent and will fix that for the 2022.09 release.
